modules/oeFax/faxserver/fax_serve.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Fax SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2019 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
```

Comment:
TODO
This script bypasses authentication, so reminder for myself to look at flow in detail.